app/shared/src/Color.js

```diff
@@ -75,6 +75,18 @@ const ColorPalette = {
   white: '#ffffff',
 };
 
+/**
+ * from RNColors
+ *      "brand":           @"#0097A9",
+        "brandSecondary":  @"#0CB3C7",
+        "buttercup":       @"#EB9D08",
+        "sun":             @"#FBAD18",
+        "backgroundGray":  @"#EDEFF2",
+        "textDark":        @"#30363D",
+        "textMedium":      @"#596066",
+        "textLight":       @"#79818A",
+ */
```

Comment:
Maybe we should remap it manually so we'll be sure that we are actually using the right name in the code and the autocomplete will work?

Reply:
Makes sense, it is not that many colors either. I will do it.